data/typelibrary/utils-1.0.0/typelib/E_STOPWATCH.fbt

```diff
@@ -1,11 +1,13 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <FBType Name="E_STOPWATCH" Comment="FB for stopping time between events">
-	<Identification Standard="61499-2" Description="Copyright (c) 2018 fortiss GmbH&#10;&#9;&#9;&#9;   2020 Johannes Kepler University Linz &#10;  &#10;This program and the accompanying materials are made &#10;available under the terms of the Eclipse Public License 2.0 &#10;which is available at https://www.eclipse.org/legal/epl-2.0/ &#10; &#10;SPDX-License-Identifier: EPL-2.0" >
+	<Identification Standard="61499-2" Description="Copyright (c) 2018 fortiss GmbH&#10;&#9;&#9;&#9;   2020 Johannes Kepler University Linz&#10; &#9;&#9;&#9;   2024 HR Agrartechnik GmbH&#10;  &#10;This program and the accompanying materials are made &#10;available under the terms of the Eclipse Public License 2.0 &#10;which is available at https://www.eclipse.org/legal/epl-2.0/ &#10; &#10;SPDX-License-Identifier: EPL-2.0" >
```

Comment:
as weith the C++ files please keep the years together: it should be:
   (c)  2018, 2020 Johannes Kepler University Linz, HR Agrartechnik GmbH
          ^         ^
          |          last year the file was changed
          The year the file was created